en/appendices/3-5-migration-guide.rst

```diff
@@ -10,7 +10,30 @@ Deprecations
 The following is a list of deprecated methods, properties and behaviors. These
 features will continue to function until 4.0.0 after which they will be removed.
 
-.. Add deprecations here.
+Deprecated Combined Get/Set Methods
+-----------------------------------
+
+In the past CakePHP has leveraged 'modal' methods that operate as provide both
```

Comment:
@markstory I think there is something wrong with this part of the sentence: 

 > methods that operate as provide both